UI.tsv

```diff
@@ -325,7 +325,7 @@ UI_20150317_000324	{@st56}Dead{/}
 UI_20150317_000325	Spell Power
 UI_20150317_000326	Defense
 UI_20150317_000327	Magic Resistance
-UI_20150317_000328	Critical Rate
+UI_20150317_000328	Critical Attack Chance
```

Comment:
To me, Critical Rate is the "Rate of which you land a critical hit". So I find it just as clear as your suggestion, just shorter. If we can determine that the units for the number is in fact a percentage rather than a frequency, I'd be happy to accept Chance instead of Rate.

Also, I think either term would be alien to anyone who hasn't played an MMO (let alone an RPG). It'd be just like Blunt, Pierce, Slash/ AoE/ etc. For such cases, they can refer to wikis/guides to learn terminologies. I don't think there's a need to simplify terminologies right now unless they're really convoluted.